### DISCUSSION TITLE
how to more GPU used when SD running in intel UHD 3.9G GPU

### DISCUSSION THREAD

**Discussion by jbtraining** [openvinotoolkit/openvino_notebooks#1355] [AI Solution Kits]
hello all,
appreciate you can see this issue.
I am using one notebook computer, which has intel UHD 3.9G GPU, to run stable diffusion weiui+roop
my question is: look like it always use CPU, very few time to run GPU.
in order to more run GPU, I also installed openvino through below link, but it look like still same. 
https://github.com/openvinotoolkit/openvino_notebooks/wiki/Windows
can you help to tell, how to more GPU used when SD running.
Thanks a lot.
Good day!

![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/146560149/45e506af-d362-4136-a433-42e892c012e6)

**Comment by brmarkus** [comment]
Can you provide more details about which Jupyter notebooks you specifically tried to run?
Some do hard-code the inference-device to use to be `CPU`, others hard-code `GPU`, some use e.g. `AUTO` or combinations of CPU and GPU like `device=CPU,GPU` (or similar).
Often you can simply change `device="CPU"` to `device="GPU"` - but there are cases where a specific neural-network model is supported on CPU only (or on GPU only) (e.g. due to the included operations or layers, or whether the model was quantized, etc.).

Example: "https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image-demo.ipynb"
It allows you to easily change between `CPU` and `GPU`, see cell `Out[6]: Dropdown(description='Device:', index=2, options=('CPU', 'GPU', 'AUTO'), value='AUTO')`

**Reply by jbtraining** [reply]
hello brmarkus, appreciate you feedback.
I am a beginner of SD, my target is use GPU for sd； I installed based on 236 readme file of below link:
https://github.com/openvinotoolkit/openvino_notebooks/wiki/Windows

but files are too big, now my hard disk is full, so cause me can't do other things.
in order to make sure my computer can work smoothly, I need remove openvino
if so, how to use GPU for sd.
and how to remove openvino from my computer.
thanks a lot.
